src/content/ui-api/management-api/creating-workload-clusters/aws/index.md

```diff
@@ -44,6 +44,7 @@ kind: Cluster
 metadata:
   generation: 1
   labels:
+    tag.provider.giantswarm.io/Environment: PROD
```

Comment:
one question regarding labels, did we ever had an agreement keeping labels in lower case?

Reply:
the reason why I'm asking because kubernetes labels are case-sensitive

Reply:
also tag.provider.giantswarm.io/Environment ->  tag.provider.giantswarm.io/environment

Reply:
well this is up to the customer. We just define the prefix `tag.provider.giantswarm.io` and they will decide if they want to use lower or uppercase. We could define some guidelines but not enforce them, right?

Reply:
fully agree, we should have some guidelines around labels. It has some implications when using label selector and stuff so customer has to be careful in case it uses those labels to select. Not blocking here and thanks for updating the docs 🚀 .

Reply:
I would also prefer giving an all lowercase example. Seems to me as a good habit to not involve uppercase/lowercase selection issues with labels.